labs/lab-2-javascript-game/elicecilgame.js

```diff
@@ -0,0 +1,59 @@
+var grantHealth = 10;
+var userHealth = 40; 
+var winCount = 0;
+
+// class User {
+//     constructor()
+// }
+
+const startGame = function() {
+    var start = prompt("Would you like to fight the Champion?");
+    if(start === "yes") {
```

Comment:
You could use `start.toLowerCase()` here to let the user say "yes" or "Yes" or "YES" or "YeS".